### ISSUE TITLE
Decide on whether to use {golem}

### ISSUE THREAD

**Issue by FrieseWoudloper** [informalr/snackman#37]
The `golem` framework makes it easier to develop large and complex Shiny apps and adhere to best practises. This issue may only be relevant if we decide to use Shiny modules (https://github.com/informalr/snackman/issues/24).

Hadley Wickham in [Mastering Shiny](https://mastering-shiny.org/) recommends using modules and `golem` for more complex Shiny apps.

> [...] if you’re making a complicated app, particularly if there are multiple authors, there are substantial advantages to making a full fledged package. If you want to do this, I recommend reading the [Engineering Shiny](https://engineering-shiny.org/) book and using the accompanying golem package.

**Comment by janclod** [comment]
I totally agree in relying on a framework like golem.

I have experimented with (here)[https://github.com/janclod/marine].

I thought that the whole point of using a framework was to free my hands from writing repetitive code and help me strucutre my project, but I found out:
* golem is not using the most recent version of shiny

...and found myself having to correct golem generated code templates.

One thing that worked very smoothly was deploying the app to shinyapps.io, but I never deployed without golem. So not sure about advantages.
